documentation/bolt.ditamap

```diff
@@ -16,9 +16,11 @@
         <topicref href="projects.md" format="markdown"/>
         <topicref href="inventory_file_v2.md" format="markdown"/>
         <topicref href="configuring_bolt.md" format="markdown"/>
-        <topicref href="using_plugins.md" format="markdown">
+        <topichead navtitle="Plugins">
```

Comment:
After reading through this stuff, I think we can move plugins out of the configuring section and just list them in the main nav. Their application is much broader than just configuration. Maybe this section can go after modules.

Reply:
Done and done! :)